Sources/Core/Services/RateObservingService.swift

```diff
@@ -102,6 +104,8 @@ final class ModernAVPlayerRateObservingService: RateObservingService {
             timer?.invalidate()
             onTimeout?()
         } else {
+            //try to force play if playback stalled
+            player.play()
```

Comment:
sorry but don't understand your point here.

did you manage to resolve issues with this ?

this service is only used to observing rate, and should stay like these. 
playing something is another job and should be done by associated controller or other service.. 
a service has one job, only one. use callback if necessary

Reply:
you are right, that was my local fix, that should not be included in this PR, will remove this change